Makefile

```
@@ -207,10 +207,14 @@ CGO_ENABLED := 0
 endif
 export CGO_ENABLED
 
+export GIT_REPO 		 ?= github.com/operator-framework/operator-controller
+export VERSION           ?= $(shell git describe --tags --always --abbrev=0)
+export VERSION_PATH 	 ?= ${GIT_REPO}/internal/version
+export GIT_COMMIT 		 ?= $(shell git rev-parse HEAD)
 export GO_BUILD_ASMFLAGS := all=-trimpath=$(PWD)
-export GO_BUILD_LDFLAGS := -s -w -X $(shell go list -m)/version.Version=$(VERSION)
 export GO_BUILD_GCFLAGS := all=-trimpath=$(PWD)
 export GO_BUILD_FLAGS :=
+export GO_BUILD_LDFLAGS  := -s -w -X '$(VERSION_PATH).version=$(VERSION)' -X '$(VERSION_PATH).gitCommit=$(GIT_COMMIT)'
```

Nit to avoid long lines (note the extra empty line at the end so that we can put `\` at the end of each line consistently).
```suggestion
export GO_BUILD_LDFLAGS  := -s -w \
    -X '$(VERSION_PATH).version=$(VERSION)' \
    -X '$(VERSION_PATH).gitCommit=$(GIT_COMMIT)' \
    
```